src/modules/comments.js

```diff
@@ -0,0 +1,38 @@
+import { baseURL, appId } from './involveapi.js';
+
+// eslint-disable-next-line consistent-return
+async function getComments(pokemonName) {
```

Comment:
- [ ] Kindly make sure to use arrow functions in all of your codebase so it's using ES6 syntax and looks semantic. it's better to not mix arrow functions with regular functions